microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -598,25 +542,112 @@ See also:
+* This checkbox is present in Microsoft Edge Stable 131.
 
 
-<!-- ------------------------------ -->
-#### Performance panel: enable server timings in the timeline
+<!-- ====================================================================== -->
+## Performance panel: enable server timings in the timeline
 <!-- checkbox has no "(?)" link pointing to this anchor wording -->
 
 _new as of Microsoft Edge Stable 129_
 
 Shows a **Server Timings** track in the **Main** section of the **Performance** tool, to display server timing events sent by your HTTP server. To learn more, see [Server timing](https://developer.mozilla.org/docs/Web/API/Performance_API/Server_timing) at MDN Web Docs.
-<!-- todo: desc -->
+<!-- todo old: desc -->
 
 See also:
 * [Introduction to the Performance tool](../evaluate-performance/index.md)
 
 Status:
-* This checkbox is present in Microsoft Edge Canary 131.
-* This checkbox is present in Microsoft Edge Stable 129.
+* This checkbox is present in Microsoft Edge Canary 133.
+* This checkbox is present in Microsoft Edge Stable 131.
+
+
+<!-- ====================================================================== -->
+## Extension storage in Application panel
+<!-- checkbox has no "(?)" link pointing to this anchor wording -->
+
+_new as of Microsoft Edge Stable 131_
+
+<!-- todo: desc -->
+
+<!-- 
+See also: 
+* []()
+-->
+
+Status:
+* This checkbox is not present in Microsoft Edge Canary 133.
+* This checkbox is present in Microsoft Edge Stable 131.
+
+
+<!-- ====================================================================== -->
+## Floating entry points for the AI assistance panel
+<!-- checkbox has no "(?)" link pointing to this anchor wording -->
+
+_new as of Microsoft Edge Canary 133_
+
+<!-- todo: desc -->
+
+<!-- 
+See also: 
+* []()
+-->
+
+Status:
+* This checkbox is present in Microsoft Edge Canary 133.
+* This checkbox is present in Microsoft Edge Stable 131.
+
+
+<!-- ====================================================================== -->
+## Performance panel: enable experimental performance insights
+
+_new as of Microsoft Edge Canary 133_
+
+<!-- todo: desc -->
+
+<!-- 
+See also: 
+* []()
+-->
+
+Status:
+* This checkbox is present in Microsoft Edge Canary 133.
+* This checkbox is present in Microsoft Edge Stable 131.
+
+
+<!-- ====================================================================== -->
+## Performance panel: enable dimming unrelated events in performance insights and search results
+
+_new as of Microsoft Edge Canary 133_
+
+<!-- todo: desc -->
+
+<!-- 
+See also: 
+* []()
+-->
+
+Status:
+* This checkbox is present in Microsoft Edge Canary 133.
+* This checkbox is not present in Microsoft Edge Stable 131.
+
+
+<!-- ====================================================================== -->
+## Performance panel: enable a switch to an alternative timeline navigation option
+
+_new as of Microsoft Edge Canary 133_
+
+<!-- todo: desc -->
```

Comment:
need desc: Performance panel: enable a switch to an alternative timeline navigation option

Reply:
This doesn't seem to work for me yet. But this is supposed to enable a switch inside the shortcut dialog, which is this dialog:
<img width="734" alt="image" src="https://github.com/user-attachments/assets/72e4276a-0c25-4348-99dc-2d19cd82e9cd">

The switch is supposed to let users choose how the main chart reacts to mouse wheel events (whether it zooms in/out, or whether it pans).

Reply:
result:
```
In the **Performance** tool, adds a **Switch to an alternative timeline navigation** control 
inside the **Help/ Keyboard shortcuts** popup.  The switch lets you choose how the main 
chart reacts to mouse wheel events (whether the mouse wheel zooms the chart in/out, or 
whether the mouse wheel pans the chart left/right).
<!-- todo: when able to access UI, update desc -->

![Alternative timeline navigation option](./index-images/alt-timeline-nav.png)
```